scripts/yast2-funcs

```diff
@@ -111,6 +111,21 @@ function set_qt_home_dir()
     fi
 }
 
+function set_qt_arabic_workaround()
+{
+    # Work around clipped Arabic text, bsc#880701
+    # Upstream bug: https://bugreports.qt.io/browse/QTBUG-41450
+    # Locale check: first of the variables wins; man 7 locale
```

Comment:
I'd prefer some `TODO` or `FIXME` tag so we can easily find this workaround by `grep` later and verify whether it's still needed.